learners/BF_measles.Rmd

```diff
@@ -0,0 +1,884 @@
+          "exposed and infected individuals ",
+          italic(R)[0], " = 15 for different age groups."
+        )
+      )
+  ) +
+  guides(
+    color = guide_legend(nrow = 2)
+  )
+```
+
+
+## Effective Reproduction Number $\mathcal{R}_{eff}$
+
+Age-structured models account for heterogeneous characteristics in the
+population, such as:
+
+-   Demographic differences in social contacts,
+-   Variations in demographic distribution, and
+-   Differences in susceptibility to infection.
+
+These factors result in different $\mathcal{R}_0$ values for each age
+group. The **effective reproduction number** $\mathcal{R}_{eff}$
+represents the overall reproduction number across all age groups while
+accounting for these differences. It is calculated using the following
+formula:
+
+[$$
+\begin{equation}
+\mathcal{R}_{eff} = \rho(C) \cdot \lambda \cdot \gamma 
+\end{equation}
+ \qquad(5)$$]{#eq-reff}
+
+Where:
+
+-   $\rho(C)$ is the **largest eigenvalue** of the contact matrix $C$,
+    which is derived from the social contact patterns, scaled by the
+    population distribution across different demographic and
+    susceptibility groups.
+
+-   $\lambda$ is the **force of infection**, which measures the rate at
+    which susceptible individuals are infected. This is distinct from
+    the effective transmission rate, often denoted as $\beta$.
+
+-   $\gamma$ represents the **infectious period**, which is the average
+    duration an individual remains infectious.
+
+The `{finalsize}`package implements
+[Equation 5](#eq-reff), and we can use it to calculate the
+$\mathcal{R}_{eff}$ for our age-structured model, as follows:
+
+
+```{r}
+r0 <-  15 #  basic reproduction number
+
+# define the number of age and susceptibility groups
+n_demo_grps <- length(demography_vector)
+n_risk_grps <- 5
+
+# In this example, risk varies across groups
+susceptibility <- matrix(
+  data = c(0.80, 0.5, 0.3, 0.2, 0.1), nrow = n_demo_grps, ncol = n_risk_grps
+)
+
+# risk does not vary within groups
+p_susceptibility <- matrix(
+  data = 1, nrow = n_demo_grps, ncol = n_risk_grps
+)
+
+# p_susceptibility rows must sum to 1.0
+p_susceptibility <- p_susceptibility / rowSums(p_susceptibility)
+
+
+reff <- finalsize::r_eff(
+  r0 = r0,
+  contact_matrix = contact_matrix,
+  demography_vector = demography_vector,
+  susceptibility = susceptibility,
+  p_susceptibility = p_susceptibility
+)
+
+reff
+```
+
+::: challenge
+### $\mathcal{R}_{eff}$ vs $\mathcal{R}_0$
+
+How different is the $\mathcal{R}_{eff}$ value from $\mathcal{R}_0$?
+:::
+
+This estimate of $\mathcal{R}_{eff}$ more accurately reflects the transmission potential and can 
+be used to estimate the **HIT** required for achieving sufficient vaccine coverage
+
+# Interventions models
+
+Interventions against measles include prevention, control, and
+treatment. Controlling its spread requires a combination of public
+health measures, such as reducing contact, isolating infectious and
```

Comment:
Are there any examples of contact reductions in response to measles outbreaks? E.g. school closures? Otherwise might feel quite hypothetical.

I don't think we talk about isolation later, so might be best to cut this or explain why not considered.

Reply:
Update: 2019 measles in Samoa would be one example where curfew introduced and schools closed: https://en.wikipedia.org/wiki/2019_Samoa_measles_outbreak

Reply:
Thanks for the example, and opt to cut isolation.